### DISCUSSION TITLE
Check mark to change from 'Added to Watchlist' to 'My Shows' for series/shows

### DISCUSSION THREAD

**Discussion by togatown** [trakt/showly#819] [Ideas / Feature Requests]
When a movie is on your Watchlist and you watch it, there is a check mark next to the "Added to Watchlist" button so you can easily mark it as watched. But, with series/shows, when they are on the watchlist, you have to remove the series/show from the watchlist and then "Add to my Shows". It would be nice to have the same check mark to change the status like with the movies

**Comment by michaldrabik** [comment]
You can already do it faster by pressing and holding an item in your Watchlist list and also from other different places in the app.

**Reply by togatown** [reply]
Thanks, never tried that!
